pypesto/ensemble/ensemble.py

```diff
@@ -571,6 +572,11 @@ def from_sample(
         remove_burn_in:
             Exclude parameter vectors from the ensemble if they are in the
             "burn-in".
+        rel_cutoff:
+            Relative cutoff. Exclude parameter vectors, for which the
+            (non-normalized) posterior value difference to the best vector is greater than
+            cutoff, i.e. include all vectors such that
+            `neglogpost(vector) <= neglogpost(max * rel-cutoff)`.
```

Comment:
If i understand the calculate_hpd correct, it is not the ones that are greater than a certain differnece, but the the `real_cutoff`-quantile (line 1324), in this case one should perhaps rename it more appropriately to `x_quantile` and also check (again line 1324) that it is <= 1?

Reply:
So basically what I meant with "difference to best vector" is that we do not have symmetric percentiles but rather cut the parameters with the lowest posterior probability. Maybe the following description makes it clearer:

"Exclude parameter vectors, which have a (non-normalized) posterior value below the defined credibility level."

If we agree to use the naming "quantile" explicitly we have to make sure that there is no confusion on the quantile calculation on parameter values vs. the calculation on the posterior probability of the joint samples in this case.